### PR TITLE
feat: add create entity data table check component 

### PR DESCRIPTION
**Pull request by pearlMat**
## Description
Entity data table check component completed.

## Related issues and discussion
#169

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [✔️ ] If you have multiple commits please combine them into one commit by squashing them.

- [✔️ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.
cedtc

### CODE REVIEW

**Comment by karantakalkar** [comment]
Hi, pearl you can do the below given to resolve errors and make your PR fit as per guidelines:

1. Correct GitHub markup use [x] to check the checklist in PR description, Related issue is #169 don't use #{169}, add Fixes: #169 in your PR description (use git commit amend)  and add (#663) (not #169) just beside your PR title.
2. use ng lint --fix=true to resolve linting errors (style/syntax errors) however you may still have to resolve jsdoc errors and operator errors yourself.
3. You have either not added or not included module which contains component declaration which is the second reason your build is failing.
4. Please refactor your code for indentations and add relevant comments.
5. check out [this](https://github.com/openMF/web-app/pull/577/files) PR to see how a component is declared and how code is commented and indented.

**Comment by pearlMat** [comment]
Hi Karan..thanks for the pointers.I want to squash my commits but i cant seem to find how to do so after running this command git rebase --interactive HEAD~5.

File: .travis.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@ deploy:
   github-token: $GITHUB_TOKEN
   keep-history: true
   on:
-    branch: master
```
Comment (karantakalkar) [inline .travis.yml]: exclude this file from your commit

File: src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  templateUrl: './create-entity-data-table-check.component.html',
+  styleUrls: ['./create-entity-data-table-check.component.scss']
+})
+export class CreateEntityDataTableCheckComponent implements OnInit {
+  /**
+   * form fields
+   */
+
+  form: FormGroup;
+  entity: string;
```
Comment (karantakalkar) [inline src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts]: add comments for each variable see reference PR in the comment

File: src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    @Inject(MAT_DIALOG_DATA) public data: any,  public fb: FormBuilder) {
+
+      this.form = fb.group({
+        entity: [this.entity, Validators.required],
+        status: [this.status, Validators.required],
+        dataTable: [this.dataTable, Validators.required],
+        product: [this.product ] });
+     }
+
+save() {
```
Comment (karantakalkar) [inline src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts]: functions should come in a single line correct for indentation.

File: src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  dataTable: string;
+  product: string;
+
+ /**
+  * @param {MatDialogRef} dialogRef Component reference to dialog.
+  * @param {any} data Provides a Create Entity Data Table CheckContext.
+  */
+  constructor(public dialogRef: MatDialogRef<CreateEntityDataTableCheckComponent>,
+    @Inject(MAT_DIALOG_DATA) public data: any,  public fb: FormBuilder) {
+
```
Comment (karantakalkar) [inline src/app/organization/create-entity-data-table-check/create-entity-data-table-check.component.ts]:
```suggestion

```

File: src/app/organization/entity-data-table-checks/entity-data-table-checks.component.ts [hunk 5]
```diff
@@ -127,4 +127,18 @@ export class EntityDataTableChecksComponent implements OnInit {
...
   }
 
+   /**
+    * Opens the create entity data table check dialog
+    */
+
+  openCreateEntityDataTableCheckDialog() {
+
+    const dialogConfig = new MatDialogConfig();
+
```
Comment (karantakalkar) [inline src/app/organization/entity-data-table-checks/entity-data-table-checks.component.ts]:
```suggestion

```

File: src/app/organization/entity-data-table-checks/entity-data-table-checks.component.ts [hunk 6]
```diff
@@ -127,4 +127,18 @@ export class EntityDataTableChecksComponent implements OnInit {
     });
   }
 
+   /**
+    * Opens the create entity data table check dialog
+    */
+
+  openCreateEntityDataTableCheckDialog() {
+
```
Comment (karantakalkar) [inline src/app/organization/entity-data-table-checks/entity-data-table-checks.component.ts]:
```suggestion

```

File: .travis.yml (hunk 1 above)
Comment (pearlMat) [reply .travis.yml]: How can i do this please? Sorry to bother you. I am a beginner..

Comment (karantakalkar) [reply .travis.yml]: check [this](https://stackoverflow.com/questions/12481639/remove-files-from-git-commit) out, also you can use VS code text editor it simplifies a lot of things

Comment (pearlMat) [reply .travis.yml]: I have followed the instructions but i am not sure it works yet. 
Do i need to wait for my PR to be granted before i work on
another issue?

Comment (pearlMat) [reply .travis.yml]: 1 have removed .travis from my commit. what else do i need to do...to move on please?

Comment (Rishabh106) [reply .travis.yml]: You do not have to completely remove the file, you have to remove the changes added by you in the file. Add the file again by copying the code from the repository and push the changes.

Comment (pearlMat) [reply .travis.yml]: @Karantakalkar please have a look. I have done the suggested changes

**Comment by karantakalkar** [comment]
Closing due to inconsistency and inactivity.

**Pull request closed without merging**